CHANGELOG.md

```diff
@@ -25,8 +25,28 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Change variable `saml_launchpad_metadata_path` to
     `saml_launchpad_metadata_url` in the `tf-modules/cumulus` Terraform module.
 
+- **CUMULUS-1446**
+  - `@cumulus/integration-tests/index.rulesList()` updated to only take
+    `rulesDirectory` parameter
+  - Remove unused `@cumulus/integration-tests/api.addCollectionApi()` function
```

Comment:
I think we need to deprecate all these functions, since they are publicly exported?